src/bundle/Resources/public/scss/fieldType/edit/_ezimageasset.scss

```diff
@@ -0,0 +1,74 @@
+.ez-field-edit--ezimageasset {
+    .ez-field-edit-preview {
+        padding: 1.25rem 0 1.25rem 1.25rem;
+
+        &__media {
+            max-width: 100%;
+            width: auto;
+            height: auto;
+            display: block;
+        }
+
+        &__details {
+            color: $ez-color-base-light;
+            font-size: .75rem;
```

Comment:
`0.75rem`, Prettier adding zero at the beginning. Please install Prettier and run on all JS and CSS files.